packages/realm-web/src/User.ts

```diff
@@ -0,0 +1,138 @@
+import type { App } from "./App";
+
+interface UserParameters {
+    app: App<any>;
+    id: string;
+    accessToken: string;
+    refreshToken: string;
+    controllerReady?: (controller: UserController) => void;
```

Comment:
`controllerReady` is not descriptive. Isn't it better like `controllerCreated` or `controllerCreatedCallback`

Reply:
I renamed to `onController`.

Reply:
👍